docs/source/markdown/podman-create.1.md

```diff
@@ -881,6 +886,37 @@ content label. Shared volume labels allow all containers to read/write content.
 The `Z` option tells Podman to label the content with a private unshared label.
 Only the current container can use a private volume.
 
+  `Overlay Volume Mounts`
+
+   The `:O` flag tells Podman to mount the directory from the host as a
+temporary storage using the Overlay file system. The containers
+are allowed to modify contents within the mountpoint and are stored in the
+container storage in a separate directory.  In Overlay FS terms the source
+directory will be the lower, and the container storage directory will be the
+upper. Modifications to the mount point are destroyed when the container
+finish executing, similar to a tmpfs mount point being unmounted.
+
+  Subsequent execution of the container will see the original source directory
+content, any changes from previous container executions no longer exists.
+
+  One use case of the `overlay` mount is sharing the package cache from the
```

Comment:
\`overlay\` vs Overlay vs overlay ... not sure if that's intended

Reply:
Fixed to lower case overlay.